src/healthservice/HealthService.py

```diff
@@ -0,0 +1,44 @@
+from fastapi import FastAPI, Query
+from .database import BaseHealthDB
+from .database.health_schema import HealthEntry, BaseHealthEntry, Health, BaseHealth
+from typing import Annotated
+
+
+class HealthService:
+    def __init__(self, app: FastAPI, database: BaseHealthDB, cfg: dict) -> None:
+        self.__app = app
+        self.__db = database
+        self.__cfg = cfg
+        
+    def configure_database(self):
+        @self.__app.on_event("startup")
+        def startup():
+            self.__db.startup()
+        
+        @self.__app.on_event("shutdown")
+        def shutdown():
+            self.__db.shutdown()
+            
+    def configure_routes(self):
+        self.__app.add_api_route("/insertHealth", self.InsertHealthEntry, methods=["POST"])
+        self.__app.add_api_route("/deleteHealth", self.DeleteHealthEntry, methods=["DELETE"])
+        self.__app.add_api_route("/getHealth", self.GetUsersLatestHealthEntry, methods=["GET"])
+        self.__app.add_api_route("/UserHealthHistory", self.GetUsersHealthEntries, methods=["GET"])
+        
+    async def InsertHealthEntry(self, health: BaseHealthEntry):
+        return self.__db.InsertHealthEntry(health) 
```

Comment:
no error handling when a error happens it is just returned to the user with a http 200 code